features/rendering.feature

```diff
@@ -154,7 +154,7 @@ Feature: Rendering
     When I run jekyll build
     Then I should get a zero exit status
     And the _site directory should exist
-    And I should see ".foo-bar {\n  color: red; }" in "_site/index.css"
+    And I should see ".foo-bar { color: red; }\n\n\/\*# sourceMappingURL=index.css.map \*\/" in "_site/index.css"
```

Comment:
Source map additions - should we check that those files are also output?

Reply:
Ideally, that is unnecessary.
The sass-converter plugin already [covers](https://github.com/jekyll/jekyll-sass-converter/blob/5d130e8791fc45f60ca7b41c1926b1ee8bf1e279/spec/scss_converter_spec.rb#L380) testing the generation and rendering of sourcemaps.